react/PostalCodeGetter.js

```diff
@@ -75,6 +75,7 @@ class PostalCodeGetter extends Component {
       default:
       case POSTAL_CODE: {
         const field = getField('postalCode', rules)
+        const shouldShowNumberKeyboard = hasPostalCodeOnlyNumber(field.mask)
```

Comment:
you don't need `hasPostalCodeOnlyNumber`

> isNaN returns true if the variable does NOT contain a valid number

```js
isNaN(123)         // false
isNaN('123')       // false
isNaN('1e10000')   // false (This translates to Infinity, which is a number)
isNaN('foo')       // true
isNaN('10px')      // true
```

```suggestion
        const shouldShowNumberKeyboard = !isNaN(field.mask)
```